### PR TITLE
Add aerobulk-python

### PR DESCRIPTION
**Pull request by jbusecke**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jbusecke** [comment]
@rabernat could you confirm that we got everything covered in terms of the license (two unchecked bullets in the top comment). Once those are checked I think this is ready to go.

**Comment by jbusecke** [comment]
Oh also @rabernat I added you as a maintainer, hope that was ok.

**Comment by rabernat** [comment]
of course! :+1:

**Comment by rabernat** [comment]
> * If static libraries are linked in, the license of the static library is packaged.

We are using the same license as the static library (GPL-3):
https://github.com/brodeau/aerobulk/blob/master/LICENSE

I hope this is correct.

**Comment by jbusecke** [comment]
@conda-forge-admin, please ping team

Please see the above discussion. With this package we provide a python wrapper for the [aerobulk](https://github.com/brodeau/aerobulk/issues/8). We currently use aerobulk as a submodule and are unsure what that means in terms of the license. Is there some guidance for packages that are wrappers for other packages by any chance? Many thanks

File: recipes/aerobulk-python/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (ocefpaf) [inline recipes/aerobulk-python/meta.yaml:1]: Ping! BTW, you can probably create recipes faster and almost merge-ready with grayskull. See the comment below.

Comment (jbusecke) [reply recipes/aerobulk-python/meta.yaml:1]: Ah yes, thanks. I meant to do that, actually. Let me remove the hints now.

**Comment by jbusecke** [comment]
@ocefpaf do you have an opinion on the issue with the license discussed above?

**Comment by ocefpaf** [comment]
> @ocefpaf do you have an opinion on the issue with the license discussed above?

That is fine for python wrappers. There are no source packages for Fortran code and the license is compatible.

The last issue for us to merge is the version number. 0.0.0 is not really valid.

**Comment by jbusecke** [comment]
Ok Ill update the pypi release to 0.1.0 and then correct it here? Thanks for the help.

**Comment by jbusecke** [comment]
Hmm I am running into issues here again. The v0.0.0 that the original version of this recipe checked out was uploaded manually, and included a binary wheel (https://pypi.org/project/aerobulk-python/0.0.0/#files). I then tried to automate the pypi release via a github action [here](https://github.com/xgcm/aerobulk-python/blob/main/.github/workflows/publish.yml), but I had to restrict it to a source distribution, which does not include the fortran code from the aerobulk repo, which we keep as a git submodule.
I believe that the issue for the current CI failures are simply missing files since the source tarball does not include the git submodule content. 

Is there a recommended way to deal with this in a conda-forge recipe? Or do I need to deal with this on the pypi upload side? 

Do I need to change the source url in the recipe to a github release instead of a pypi tarball? I might also be missing something else entirely.

**Comment by jbusecke** [comment]
After looking at it with fresh eyes, that makes total sense. I think I have it figured out now. Thank you very much for the pointer @ocefpaf.

**Pull request merged**